django/db/models/fields/__init__.py

```diff
@@ -1892,40 +1892,15 @@ def formfield(self, **kwargs):
 
 
 class IPAddressField(Field):
-    empty_strings_allowed = False
-    description = _("IPv4 address")
-    system_check_deprecated_details = {
+    system_check_removed_details = {
         'msg': (
-            'IPAddressField has been deprecated. Support for it (except in '
-            'historical migrations) will be removed in Django 1.9.'
+            'IPAddressField has been removed except for support in '
+            'historical migrations.'
         ),
         'hint': 'Use GenericIPAddressField instead.',
-        'id': 'fields.W900',
+        'id': 'fields.E900',
     }
 
-    def __init__(self, *args, **kwargs):
-        kwargs['max_length'] = 15
-        super(IPAddressField, self).__init__(*args, **kwargs)
-
-    def deconstruct(self):
```

Comment:
this should probably stay, as we don't want `max_length` to suddenly show up somewhere in between states.